assembly/src/main/resources/kubernetes.cfg

```diff
@@ -0,0 +1,5 @@
+#
+# Label selector used to idenfity Cellar nodes in Kubernetes cluster
+#
+pod.label.key = name
+pod.label.value = cellar
```

Comment:
I'm not sure but I think, sometimes I've seen strange effects especially with txt files that also needed some filtering. Might be no real issue here, I just noticed it. The signs looked a bit like a warning. And Windows systems might be affected.